src/ResourceManager/Sql/Commands.Sql/Failover Group/Cmdlet/NewAzureSqlFailoverGroup.cs

```diff
@@ -63,10 +65,12 @@ public class NewAzureSqlFailoverGroup : AzureSqlFailoverGroupCmdletBase
         /// <summary>
         /// Gets or sets the grace period with data loss for the Sql Azure Failover Group.
         /// </summary>
+        [Alias("GracePeriodWithDataLossHour")]
         [Parameter(Mandatory = false,
             HelpMessage = "The window of grace period that we tolerate with data loss during a failover operation for the failover group.")]
+        [Obsolete("This parameter is only for backwards compatibility; User should use 'GracePeriodWithDataLossHour' instead.")]
```

Comment:
Is this obsolete attribute correct? I thought that you changed Hour to Hours. So Hour should be obsolete, not Hours.